admin/class-wpfep-system-status.php

```diff
@@ -50,57 +50,57 @@ public static function status_report()
 			<div class="wrap">
-				<h2><?php esc_attr_e('System Status', 'wpfep'); ?></h2>
+				<h2><?php esc_attr_e('System Status', 'wp-front-end-profile'); ?></h2>
 				<div class="notice notice_system_status_wpfep">
-					<p><?php esc_attr_e('Please copy and paste this information in your ticket when contacting support:', 'wpfep'); ?> </p>
-					<p class="submit"><a href="javascript:void" class="button-primary debug-report"><?php esc_attr_e('Get system report', 'wpfep'); ?></a></p>
+					<p><?php esc_attr_e('Please copy and paste this information in your ticket when contacting support:', 'wp-front-end-profile'); ?> </p>
+					<p class="submit"><a href="javascript:void" class="button-primary debug-report"><?php esc_attr_e('Get system report', 'wp-front-end-profile'); ?></a></p>
 					<div id="debug-report">
 						<textarea readonly="readonly"></textarea>
-						<p class="submit"><button id="copy-for-system-support" class="button-primary" href="javascript:void"><?php esc_attr_e('Copy for Support', 'wpfep'); ?></button></p>
-						<p class="copy-error hidden"><?php esc_attr_e('Copying to clipboard failed. Please press Ctrl/Cmd+C to copy.', 'wpfep'); ?></p>
+						<p class="submit"><button id="copy-for-system-support" class="button-primary" href="javascript:void"><?php esc_attr_e('Copy for Support', 'wp-front-end-profile'); ?></button></p>
+						<p class="copy-error hidden"><?php esc_attr_e('Copying to clipboard failed. Please press Ctrl/Cmd+C to copy.', 'wp-front-end-profile'); ?></p>
 					</div>
 				</div>
 				<table class="wpfep-status-table widefat">
 					<thead>
 					<tr>
-						<th colspan="3" data-export-label="WordPress Environment"><h2><?php esc_attr_e('WordPress Environment', 'wpfep'); ?></h2></th>
+						<th colspan="3" data-export-label="WordPress Environment"><h2><?php esc_attr_e('WordPress Environment', 'wp-front-end-profile'); ?></h2></th>
 					</tr>
 					</thead>
 					<tbody>
 					<tr>
-						<td data-export-label="<?php esc_attr_e('Home URL', 'wpfep'); ?>"><?php esc_attr_e('Home URL', 'wpfep'); ?>:</td>
+						<td data-export-label="<?php esc_attr_e('Home URL', 'wp-front-end-profile'); ?>"><?php esc_attr_e('Home URL', 'wp-front-end-profile'); ?>:</td>
 						<td><?php echo esc_html($environment['home_url']); ?></td>
 					</tr>
 					<tr>
-						<td data-export-label="<?php esc_attr_e('Site URL', 'wpfep'); ?>"><?php esc_attr_e('Site URL', 'wpfep'); ?>:</td>
+						<td data-export-label="<?php esc_attr_e('Site URL', 'wp-front-end-profile'); ?>"><?php esc_attr_e('Site URL', 'wp-front-end-profile'); ?>:</td>
 						<td><?php echo esc_html($environment['site_url']); ?></td>
 					</tr>
 					<tr>
-						<td data-export-label="<?php esc_attr_e('wpfep version', 'wpfep'); ?>"><?php esc_attr_e('wpfep version', 'wpfep'); ?>:</td>
+						<td data-export-label="<?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>"><?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>:</td>
```

Comment:
@mfaizanrasool `<?php esc_attr_e('wp-front-end-profile version', 'wp-front-end-profile'); ?>">` should be `<?php esc_attr_e('WP Frontend Profile version', 'wp-front-end-profile'); ?>">`

Reply:
Mentioned changes are Fixed.